content/oci/oci.go

```diff
@@ -238,6 +238,38 @@ func (s *Store) Resolve(ctx context.Context, reference string) (ocispec.Descript
 	return desc, nil
 }
 
+func (s *Store) Untag(ctx context.Context, reference string) error {
+	s.sync.RLock()
+	defer s.sync.RUnlock()
+
+	if reference == "" {
```

Comment:
Do the check before locking?

Reply:
I prefer doing the check after locking, since this ensures style consistency with `Tag()` and other functions.